modules/server/src/topics/actionkeybindtopic.cpp

```diff
@@ -34,12 +34,56 @@ using nlohmann::json;
 
 namespace openspace {
 
-bool ShortcutTopic::isDone() const {
+bool ActionKeybindTopic::isDone() const {
     return true;
 }
 
-std::vector<nlohmann::json> ShortcutTopic::shortcutsJson() const {
-    std::vector<nlohmann::json> json;
+nlohmann::json jsonAction(const interaction::Action& action) {
+    // Convert to std as nlohmann doesn't understand glm
+    nlohmann::json colorJson;
+    if (action.color.has_value()) {
+        glm::ivec4 color = action.color.value();
```

Comment:
What is the value range for action.color.value() 0-1 or 0-255? Since we are converting from float to int here won't this cause issues?

Reply:
eg. (0.1f, 0.5, 1.0f, 1.0f) => (0, 0 ,1, 1) => (0, 0, 255,255) ?

Reply:
Can also swap the order and do multiplication first followed by conversion to ivec4 for nicer values on the frontend

Reply:
As long as we do the conversion explicitly, either works. :) 

However, see my other comment. I personally think we should do the conversion to [0,255] values _on the frontend_, when we set the value that CSS needs

Reply:
In OpenSpace we always represent color in range [0,1], so to be consistent, that's the data we should send